wiki/Tournaments/Chinese mania 4K Tournament 2017/zh.md

```diff
@@ -0,0 +1,208 @@
+| 直播裁判团队 | [![CN]aiyulu](https://osu.ppy.sh/u/189617)、[![CN]Rasis](https://osu.ppy.sh/u/1209363)、[![CN]Ookura Asahi](https://osu.ppy.sh/u/4786951) |
+| 解说团队 | [![CN]hans1999](https://osu.ppy.sh/u/6679329)、[![CN]Scslag](https://osu.ppy.sh/u/6705256)、[![CN]Rasis](https://osu.ppy.sh/u/1209363)、[![CN]644520858](https://osu.ppy.sh/u/5507069) |
+
+## 比赛直播间
+
+Bilibili直播平台：[aiyulu](http://live.bilibili.com/10183)(主直播)、[Ookura Asahi](http://live.bilibili.com/272810)(副直播)、[Rasis](http://live.bilibili.com/5333)(备用)。  
+Twitch直播平台：[aiyulu](http://live.bilibili.com/10183)。
+
+## 对阵表（小组赛开始后更新）
+
+## 比赛预告
+
+### 测试赛
+
+6月17日 15:00(UTC+8)  新锐队 vs 元老队 (4v4）
+
+元老队队员：**gpf19851210** IDSxianyu Cideisbiantai Scslag hans1999 w765333805
+
+新锐队队员：**Moting** Mof liberavi prelude8792916 qiluo wjh123 xxxx
+
+## 比赛图池
+
+### 测试赛图池
+[点此下载全部](https://1drv.ms/u/s!AuGED5b9f0ulnyuYMI1MJhnTjx9g)  
+[Shiki – Endless Dream (pompom’s Another)](https://osu.ppy.sh/b/601737)  
+[sakuzyo – Refel (Liberation)](https://osu.ppy.sh/b/1178870)  
+[-45 – EXILE (Anotehr_SideA)](https://osu.ppy.sh/b/815021)  
+[seleP – Scarlet Rose (Scarlet Rose)](https://osu.ppy.sh/b/1228278)  
+[Polyphony – Adrift (Marine)](https://osu.ppy.sh/b/658126)  
+[LeaF – I (_UJ’s Another)](https://osu.ppy.sh/b/928341)  
+[DJ Sharpnel – Cyber Induction （IcyWorld）](https://osu.ppy.sh/b/1021178)  
+[Helblinde – Rewrite Nightmare (Experiment)](https://osu.ppy.sh/b/719525)  
+[Helblinde – Memoria (Original Mix) (LeiN-‘s SC)](https://osu.ppy.sh/b/1101044)  
+[DET & Silantra – Luv Song (<3)](https://osu.ppy.sh/b/602841)  
+[Kobaryo – Pumpin’ Junkies (Kobaryo’s FTN-Remix) (Arcwin’s Maxima)](https://osu.ppy.sh/b/644265)  
+[A-Teens – Gimme! Gimme! Gimme! (Wafles’s Beginner 24)](https://osu.ppy.sh/b/1010130)  
+[senya – Zenaku no Itadaki ni Aru Shinjitsu ( EXH （LN))](https://osu.ppy.sh/b/1050840)  
+[96Neko – Paintings? Oh, yeah. (PaiLNfull)](https://osu.ppy.sh/b/1103346)  
+[LeaF – 4th smile (Prosperity)](https://osu.ppy.sh/b/1247978)  
+[Da Tweekaz – Wodka (short Ver.) (4K MX(Hard SV Edit.))](https://osu.ppy.sh/b/800952)
+
+TB:  
+[Camellia – Lunatic Rough Party!! (Long Ver.)（Lunatic）](https://osu.ppy.sh/b/646319)
+
+## 比赛回顾
+
+## 规则集
+
+### 赛事规则 
+
+1. **2017年华人osu!mania 4K大赛**(以下均简写为**CMT 4K 2017**)为华人区个人比赛，是由CMT团队举办的第二届华人osu!mania 4K大赛。比赛所用地图均为osu!mania 4K专用地图(4K专谱)。
+2. 每场比赛的地图库将于比赛前一周的周末由地图库管理员放出，图库内包含若干张Free-Mod地图:  
+	&emsp;a) 其中的Tie-breaker（决胜）地图仅用于决胜局比赛。  
+	&emsp;b) 所有地图均在Free-mod池下。
+3. 比赛日程将由组委会讨论公布。(组委会名单见公告)
+4. 若工作人员与统计员均不能就位，比赛将会被推迟。
+5. 比赛中如出现 Failed 情况，则视为负手。
+6. 选手可以使用自己的皮肤与视频设置。
+7. 如果出现平局情况，则该场比赛视作无效。
+8. 当选手掉线时，如果能够继续完成比赛，则应当在比赛结束后(使用 Shift+F12)立刻上传分数界面截图并提供网址。
+9. 如果选手因为掉线直接跳出 mp 房间界面而未能完成该图，则按以下方法处理：  
+	&emsp;a) 在比赛开始 30 秒内发生的，应予以重赛。  
+	&emsp;b) 否则视作失败（Failed）。
+10. 除非某张地图的比赛结果被列为无效，否则任何地图不能两次被用于同场比赛。
+11. 如果比赛选手因为特殊原因无法出席，最多允许等待 10 分钟，十分钟后做视为弃权。
+12. 选手自行确保比赛游戏过程中其游戏与操作系统的流畅性，例如以下情况不能重赛（包括但不限于）：  
+	- 网络原因导致的延迟、卡顿现象。   
+	- 系统弹出其他应用的通知或窗口。  
+	- 硬件故障。  
+	- 其他原因。
```

Comment:
ASG styling. Use spaces instead of tabs and the child list hyphen should align with the parent's first character.

```Markdown
12. 选手自行确保比赛游戏过程中其游戏与操作系统的流畅性，例如以下情况不能重赛（包括但不限于）：  
      - 网络原因导致的延迟、卡顿现象。
      - 系统弹出其他应用的通知或窗口。  
      - 硬件故障。  
      - 其他原因。
```